dragonfly_schema/cli/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+Command Line Interface (CLI) entry point for dragonfly-schema.
+
+Note:
+    Do not import this module in your code directly. For running the commands,
+    execute them from the command line or as a subprocess (e.g.
+    ``subprocess.call(['dragonfly-schema', 'viz'])``)
+
+Honeybee is using click (https://click.palletsprojects.com/en/7.x/) for creating the CLI.
+"""
+
+try:
+    import click
+except ImportError:
+    raise ImportError('click module is not installed.')
+
+import sys
+import os
+import logging
+import json
+
+from dragonfly_schema.model import Model
+
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands for validating JSON files.')
+def main():
+    pass
+
+
+@main.command('viz')
+def viz():
+    """Check if dragonfly-schema is flying!"""
+    click.echo('viiiiiiiiiiiiizzzzzzzzz!')
+
+
+@main.command('validate-model')
+@click.argument('model-json')
+def validate_model(model_json):
+    """Validate a Model JSON file against the Honeybee schema.
+    \b
+    Args:
+        model-json: Full path to a Model JSON file.
+    """
+    try:
+        assert os.path.isfile(model_json), 'No JSON file found at {}.'.format(model_json)
+    
+        # validate the Model JSON
+        click.echo('Validating Model JSON ...')
+        Model.parse_file(model_json)
+        click.echo('Pydantic validation passed.')
+        try:
+            import dragonfly.model as df_model
```

Comment:
Isn't dragonfly a dependency for this module? If it is then you should not need this check.

Reply:
I see that it is not a dependency. I think we should make it a dependency at least for the `cli`.